html/ImdbHtml.html

```diff
@@ -0,0 +1,69 @@
+
+<html>
+	
+	<head>
+	
+	<link type="text/css" rel="stylesheet" href="‪C:\Users\Gilron\Desktop\ImdbWebApplication\CSS\ImdbCSS.css" MEDIA="screen" />

```

Comment:
@oregilron Follow [google formatting rule](https://google.github.io/styleguide/htmlcssguide.xml?showone=General_Formatting#General_Formatting) for HTML